test/framework/src/main/java/org/elasticsearch/test/AbstractChunkedSerializingTestCase.java

```diff
@@ -24,14 +25,22 @@
     protected AbstractXContentTestCase.XContentTester<T> createXContentTester() {
         return chunkedXContentTester(
```

Comment:
Is there a way we can get an assertion about the chunk count in here? I think that's an important component of these tests.

Reply:
I bet we could set up an abstract test for this across all implementations with a little refactoring. Maybe do that in a follow-up? Not sure how much effort it will be but it shouldn't take too long. It's just that this is already so large.